### DISCUSSION TITLE
New feed not in chronological order

### DISCUSSION THREAD

**Discussion by asmaloney** [community/community#66176] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

For some reason the feed is out of order, which makes it pretty useless.

**Comment by groue** [comment]
Agree. How far do I need to scroll in order to find something *fresh*? Impossible to tell with a non-chronological feed. Should I just not scroll at all, and rely on other communication channels?

**Comment by zephyrtronium** [comment]
The message explaining the new feed claims "you can customize your feed exactly how you like it," but for me that includes chronological ordering, which is not an option. For the most part (with exceptions that already have several dozen other threads), I don't mind the other changes, but this in particular makes the feed much less useful for me.

**Comment by ChaoticRoman** [comment]
I completely agree with the original post. One of the main reasons I check my GitHub feed is to keep up-to-date with the latest activities in my starred projects and from developers I follow. The algorithmic feed makes it difficult to get a chronological sense of what's new and important. An option to toggle between the two types of feeds would be a great compromise and make the platform even more user-friendly for those of us who prefer a chronological view.
